daemon/volumes.go

```diff
@@ -103,6 +103,9 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 
 		for _, m := range c.MountPoints {
+			if _, ok := mountPoints[m.Destination]; ok {
```

Comment:
I wonder similar checking about `mountPoints` is needed for this loop (`for _, cfg := range hostConfig.Mounts {}`) as well: https://github.com/yongtang/docker/blob/d231e33926a58a11974bbdd9a038e79fac2217df/daemon/volumes.go#L164-L206